tensorflow/python/ops/list_ops.py

```diff
@@ -248,6 +248,7 @@ def _TensorListFromTensorGrad(op, dlist):
 @ops.RegisterGradient("TensorListGetItem")
 def _TensorListGetItemGrad(op, ditem):
   """Gradient for TensorListGetItem."""
+  print("---GetItemGrad---")
```

Comment:
Looks like these debug prints got pulled in.

Reply:
Fixed